### DISCUSSION TITLE
How do I continue playing when Close-Button was clicked?

### DISCUSSION THREAD

**Discussion by ProfP30** [smplayer-dev/smplayer#118] [General]
Hi,

How do I continue playing when Close-Button was clicked?
I can see a systray Icon when clicked on the "X" mark (Close-Button), but playing interrupted.
Is there a setting (searched a thousand times) or PlugIn for this?

System: Manjaro
SMPlayer Version: 21.1.0
Qt Version: 5.15.2
MPV Version: 0.33.1

Regards

**Comment by smplayer-dev** [comment]
Click on the smplayer icon in the system tray and then press the play button.

**Reply by ProfP30** [reply]
Ok, thanks.

So there is no way to stop SMPlayer from interrupting the current playing when I click on the close-button, correct?

**Reply by smplayer-dev** [reply]
The close button closes the window, why would playback continue after the window is closed?
Use the minimize button instead of close.
